topic04-git-introduction/book-git-tutorial/05.Forking.md

```diff
@@ -0,0 +1,15 @@
+# Forking on GitHub
+
+## What is forking?
+
+To fork a repository means to get your own version/copy of the repository, and to be able to edit it without affecting the original repo (repository). 
+
+## How to fork?
+
+. You first go onto someone’s github profile
+. Click on one of their repositories 
+. Then click the fork button,
+. Which is usually located below your profile picture dropdown
```

Comment:
Why is this on a separate line and have a capital "W"

Reply:
@AdamLalor well spotted. I took the content from the doc without review. Will update it thanks

Reply:
@AdamLalor updated if you have time to review again please